PlayerControls/sources/DefaultControlsViewController.swift

```diff
@@ -660,6 +660,16 @@ public final class DefaultControlsViewController: ContentControlsViewController
             showBrandedContentButton.layer.cornerRadius = showBrandedContentButton.frame.height / 2
             showBrandedContentButton.setTitle(currentUIProps.brandedContentTitle,
                                               for: .normal)
+            if nextUIProps.isBrandedContentClickable {
```

Comment:
Let's avoid this if. 
`let image = nextUIProps.isBrandedContentClickable ? UIImage(named: "open-web-page") : nil`

Reply:
Ok